runtime/src/iree-amd-aie/aie_runtime/iree_aie_runtime.h

```diff
@@ -222,6 +222,10 @@ struct AMDAIEDeviceModel {
   /// aie-rt for whatever reason. Make sure the parameters can't be retrieved in
   /// another way before adding new fields to this struct.
   struct AMDAIEDeviceConfig {
+    /// Set default addressing granularity to 32 bits as this is the value for
+    /// all current architecture versions.
+    uint8_t addressGranularityInBits{32};
```

Comment:
yea so this is unncessary today right? is the idea to plan ahead? i don't remember - is it shorter in aie4 or no?

Reply:
Yeah, it's mainly to avoid hardcoding this value everywhere else, or hide it inside the getter. I am trying to put all configs that can't be retrieved from aie-rt in this place.